frontend/src/screens/balance.vue

```diff
@@ -137,6 +145,8 @@ export default {
           backgroundClass: " bg-warning",
           message: "notEnoughInfoForBalance"
         };
+        this.existsCurrentPeriod = period.length !== 0 && period !== undefined;
```

Comment:
Debería chequearse si existe period primero y luego si tiene length
```suggestion
        this.existsCurrentPeriod = period && period.length;
```